mkdocs.yml

```diff
@@ -20,6 +20,7 @@ pages:
 - Getting Started:
   - iOS:
     - Setup: getting-started/ios/setup.md
+    - _Swift: getting-started/ios/fastlane-swift.md
```

Comment:
Does the `_` make it not visible for now?

Reply:
Yup!

Reply:
Nice 👍